sdcm/cluster.py

```diff
@@ -472,7 +472,7 @@ def install_cassandra_harry(self):
             """))
         self.remoter.run(shell_script_cmd("""\
             rm -rf cassandra-harry
-            git clone -b scylla-branch https://github.com/amoskong/cassandra-harry
+            git clone -b scylla-branch https://github.com/apache/cassandra-harry
             cd cassandra-harry
             make mvn
```

Comment:
I think we should target specific commit instead of HEAD.
```suggestion
            git clone https://github.com/apache/cassandra-harry
            cd cassandra-harry
            git checkout 26bb6696ba1b18ff5c062d780a9360e750691052
            make mvn
```